### PR TITLE
API-7864: Document new filter

### PR DESCRIPTION
**Pull request by Adimus11**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7864-document-new-flter-for-list-customers)
- [Jira](https://livechatinc.atlassian.net/browse/API-7864)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content

### CODE REVIEW

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -54,6 +54,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - The **List Customers** ([Web](/messaging/agent-chat-api/v3.3/#list-customers) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#list-customers)) method:
   - has a new parameter, `sort_by`.
   - has a new filter, `include_customers_without_chats`.
+  - has a new filter, `chat_group_ids`.
```
Comment (litwinow) [inline content/messaging/agent-chat-api/changelog/index.mdx:57]:
```suggestion
  - has new filters: `include_customers_without_chats` and `chat_group_ids`.
```

File: content/messaging/agent-chat-api/index.mdx [hunk 2]
```diff
@@ -2255,6 +2258,15 @@ There's only one value allowed for a single filter.
...
 Dates are represented in ISO 8601 format with microseconds resolution, e.g. `2017-10-12T15:19:21.010200+01:00` in specific timezone or `2017-10-12T14:19:21.010200Z` in UTC.
 
+**\*\*\*\*\*\*)**
+
+`<integer_filter_type>` can take the following values :
+
+- `values` (`int[]` - an array of integers)
+- `exclude_values` (`int[]` - an array of integers)
+
+There's only one value allowed for a single filter.
```
Comment (litwinow) [inline content/messaging/agent-chat-api/index.mdx:2268]: merge with `<string_filter_type>`, there are too many stars already :(

Comment (litwinow) [reply content/messaging/agent-chat-api/index.mdx:2268]: also, why it's described in v3.2?

File: content/messaging/agent-chat-api/v3.3/index.mdx [hunk 3]
```diff
@@ -2271,6 +2272,15 @@ There's only one value allowed for a single filter.
 
 Dates are represented in ISO 8601 format with microseconds resolution, e.g. `2017-10-12T15:19:21.010200+01:00` in specific timezone or `2017-10-12T14:19:21.010200Z` in UTC.
 
+**\*\*\*\*\*\*)**
+
+`<integer_filter_type>` can take the following values :
```
Comment (litwinow) [inline content/messaging/agent-chat-api/v3.3/index.mdx:2277]: as above with the merge

**Review by litwinow — CHANGES_REQUESTED** [review]

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 4]
```diff
@@ -53,8 +53,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - The **customer_visit_ended** push was replaced by the [**customer_left**](/messaging/agent-chat-api/v3.3/rtm-reference/#customer_left) and [**customer_transferred**](/messaging/agent-chat-api/v3.3/rtm-reference/#customer_transferred) pushes.
 - The **List Customers** ([Web](/messaging/agent-chat-api/v3.3/#list-customers) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#list-customers)) method:
   - has a new parameter, `sort_by`.
-  - has a new filter, `include_customers_without_chats`.
-  - has a new filter, `chat_group_ids`.
+  - has a new filter, `include_customers_without_chats` and `chat_group_ids`.
```
Comment (litwinow) [inline content/messaging/agent-chat-api/changelog/index.mdx:56]:
```suggestion
  - has new filters: `include_customers_without_chats` and `chat_group_ids`.
```

**Review by litwinow — APPROVED** [review]

File: content/messaging/agent-chat-api/v3.3/index.mdx [hunk 5]
```diff
@@ -2247,6 +2248,11 @@ When sorting by fields other than `created_at`, the entries with identical value
 - `values` (`string[]` - an array of strings)
 - `exclude_values` (`string[]` - an array of strings)
 
+`<integer_filter_type>` can take the following values :
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/index.mdx:2251]:
```suggestion
`<integer_filter_type>` can take the following values:
```

File: content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx [hunk 6]
```diff
@@ -2516,6 +2517,11 @@ When sorting by fields other than `created_at`, the entries with identical value
 - `values` (`string[]` - an array of strings)
 - `exclude_values` (`string[]` - an array of strings)
 
+`<integer_filter_type>` can take the following values :
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx:2520]:
```suggestion
`<integer_filter_type>` can take the following values:
```

**Review by litwinow — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**